internal/fdv2proto/event_to_storable_item.go

```diff
@@ -0,0 +1,59 @@
+package fdv2proto
+
+import (
+	"github.com/launchdarkly/go-server-sdk/v7/internal/datakinds"
+	"github.com/launchdarkly/go-server-sdk/v7/subsystems/ldstoretypes"
+)
+
+// ToStorableItems converts a list of FDv2 events into a slice of collections, which is suitable for
+// passing into a store.
+func ToStorableItems(events []Event) []ldstoretypes.Collection {
+	flagCollection := ldstoretypes.Collection{
+		Kind:  datakinds.Features,
+		Items: make([]ldstoretypes.KeyedItemDescriptor, 0),
```

Comment:
Do you think it's worth reserving some capacity here to prevent the additional allocations?

Reply:
We don't know the size up front, although we could calculate it.  I think I'd rather let Go's allocation strategy do its thing and if it's a problem we can fix it.